### PR TITLE
feat: keyset pag. sort direction option added with asc default

### PR DESCRIPTION
**Pull request by kelkarajay**

### CODE REVIEW

File: pagination/keysetpagination/paginator.go [hunk 1]
```diff
@@ -41,6 +42,13 @@ func (p *Paginator) IsLast() bool {
 	return p.isLast
 }
 
+func (p *Paginator) SortDirection() string {
```
Comment (zepatrik) [inline pagination/keysetpagination/paginator.go:45]: We cannot really change the sort direction, as we would have to flip the comparison as well. Let's just support ascending for now. Other use-cases are still possible by building the query yourself form the paginator properties.

File: pagination/keysetpagination/paginator.go [hunk 2]
```diff
@@ -62,7 +70,8 @@ func Paginate[I Item](p *Paginator) pop.ScopeFunc {
 	return func(q *pop.Query) *pop.Query {
 		return q.
 			Limit(p.Size()+1).
-			Where(fmt.Sprintf("%s > ?", id), p.Token())
+			Where(fmt.Sprintf(`"%s" > ?`, id), p.Token()).
+			Order(fmt.Sprintf(`"%s" %s`, id, p.SortDirection()))
```
Comment (zepatrik) [inline pagination/keysetpagination/paginator.go:74]:
```suggestion
			Where(fmt.Sprintf(`%q > ?`, id), p.Token()).
			Order(fmt.Sprintf(`%q ASC`, id))
```

This automatically adds double-quotes :wink:

**Review by zepatrik — COMMENTED** [review]
Just some minor remarks :wink:

**Review by zepatrik — APPROVED** [review]

File: pagination/keysetpagination/paginator.go [hunk 3]
```diff
@@ -62,7 +62,8 @@ func Paginate[I Item](p *Paginator) pop.ScopeFunc {
 	return func(q *pop.Query) *pop.Query {
 		return q.
 			Limit(p.Size()+1).
-			Where(fmt.Sprintf("%s > ?", id), p.Token())
+			Where(fmt.Sprintf(`%q > ?`, id), p.Token()).
+			Order(fmt.Sprintf(`%q ASC`, id))
```
Comment (aeneasr) [inline pagination/keysetpagination/paginator.go:66]: This should be:

```
			Where(fmt.Sprintf("`%s` > ?", id), p.Token()).
			Order(fmt.Sprintf("`%s` ASC", id))
```

Comment (aeneasr) [reply pagination/keysetpagination/paginator.go:66]: If it doesn't work across databases we will have to add a switch statement for the dialect! I think pop also has an escape helper if I'm not mistaken.

**Pull request merged**